values.yaml

```diff
@@ -387,3 +387,116 @@ connectInject:
     # configured proxy.
     proxyDefaults: |
       {}
+
+# Mesh Gateways enable Consul Connect to work across Consul datacenters.
+meshGateway:
+  # If mesh gateways are enabled, a Deployment will be created that runs
+  # gateways and Consul Connect will be configured to use gateways.
+  # See {todo doc link} for more information.
+  # Requirements: consul >= 1.6.0 and consul-k8s >= 0.8.2 if using global.bootstrapACLs.
+  enabled: false
+
+  # Which mode the gateway should run in. Can be either 'remote' or 'local'.
+  # If set to remote, Connect traffic to services in other datacenters will
+  # be sent directly to the other datacenter's gateway.
+  # If set to local, Connect traffic to services in other datacenters will
+  # be proxied via the local gateway. Use 'local' if services cannot directly
+  # connect to the other datacenter's gateway and so must proxy through the local
+  # gateway, i.e. if only the gateways can communicate between datacenters.
+  mode: remote
+
+  # Number of replicas for the Deployment.
+  replicas: 2
+
+  # What gets registered as wan address for the gateway.
+  wanAddress:
+    # Port that gets registered.
+    port: 443
+
+    # If true, each Gateway Pod will advertise its NodeIP
+    # (as provided by the Kubernetes downward API) as the wan address.
+    # This is useful if the node IPs are routable from other DCs.
+    # The port key will be used as the port for the wan address.
+    useNodeIP: true
+
+    # If true, each Gateway Pod will advertise its NodeName
+    # (as provided by the Kubernetes downward API) as the wan address.
+    # Again, this is useful if the node names are DNS entries that are
+    # routable from other DCs.
+    # The port key will be used as the port for the wan address.
+    # Must set useNodeIP to false.
+    # useNodeName: false
+
+    # If set, this will be the host of the wan address for all gateways.
```

Comment:
I'm having a hard time parsing "will be the host of the wan address for all gateways." Is there another way to phrase this to make it clearer?

Reply:
Changed to: "# If set, each gateway Pod will use this host as its wan address."